### ISSUE TITLE
DeprecationWarning: "@coroutine" decorator is deprecated

### ISSUE THREAD

**Issue by delfick** [horazont/aioopenssl#12]
Hello,

In Python3.8 my pytest tests give me 

```
.venv/lib/python3.8/site-packages/aioopenssl/__init__.py:726
  ~/project/.venv/lib/python3.8/site-packages/aioopenssl/__init__.py:726: DeprecationWarning: "@coroutine" decorator is deprecated since Python 3.8, use "async def" instead
    def create_starttls_connection(

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

**Comment by horazont** [comment]
@delfick Thanks for the note.

I’ll see that I port it to async def for the next release!

**Comment by delfick** [comment]
awesome! thanks :)

**Comment by horazont** [comment]
Fixed via #14.

**Comment by delfick** [comment]
awesome, thanks!

**Comment by delfick** [comment]
I just realised you released 0.6.0 the other day with this fix. Thanks!
